.github/workflows/ci.yml

```diff
@@ -17,7 +17,8 @@ jobs:
             - uses: actions/setup-java@v2
               with:
                   distribution: 'adopt'
-                  java-version: 11
+                  java-version: 17
```

Comment:
Suggesting use `zulu` instead of `adopt`,
https://api.adoptopenjdk.net/

Reply:
@Goooler What's the benefit? I thought `zulu` was best for M1 Macs, though I don't think the Github Mac runners are M1.

Reply:
As jake recommending this.

Reply:
Sounds good to me :)